app/decorators/video_decorator.rb

```diff
@@ -0,0 +1,12 @@
+class VideoDecorator < Draper::Decorator
+  delegate_all
+
+  def show_rating
+    average_rating ? "#{show_average_rating}/5.0" : "N/A"
+  end
+
+  private
+  def show_average_rating
```

Comment:
I'm specifically talking about the private method, in case you misunderstood. Also, you can decorate, or override, model methods too. You can do something like:

```ruby
def average_rating
  object.average_rating ? "#{rounded_average_rating}/5.0" : "N/A"
end

private

def rounded_average_rating
  object.average_rating.round(1)
end
```

As far as the confusion you worry about, that is something that you have to decide on yourself. It is definitely a possibility.